modules/overview-of-user-types-and-permissions.adoc

```diff
@@ -44,19 +44,22 @@ endif::[]
 
 Optionally, if you want to restrict access to your {productname-short} deployment, you can create specialized user groups for users and administrators.
 
+ifdef::upstream,self-managed[]
 If you decide to restrict access, and you already have user groups defined in your configured identity provider, you can add these user groups to your {productname-short} deployment. If you decide to use specialized user groups without adding these groups from an identity provider, you must create the groups in {openshift-platform} and then add users to them.
+endif::[]
+ifdef::cloud-service[]
```

Comment:
Just wanted to double-check: the `ifdef::cloud-service[]` statement opens on line 50 but doesn't seems to close until line 64. Is it our intent for everything between those lines to be only for the cloud service?